lib/stream-chat/client.rb

```diff
@@ -283,6 +285,30 @@ def get_export_channel_status(task_id)
       get("export_channels/#{task_id}")
     end
 
+    def revoke_tokens(before)
+      before = before.iso8601 if before.instance_of?(DateTime)
+      update_app_settings({ 'revoke_tokens_issued_before' => before })
+    end
+
+    def revoke_user_token(user_id, before)
+      revoke_users_token([user_id], before)
+    end
+
+    def revoke_users_token(user_ids, before)
+      before = before.iso8601 if before.instance_of?(DateTime)
+
+      updates = []
+      user_ids.each do |user_id|
+        updates.push({
+                       'id' => user_id,
+                       'set' => {
+                         'revoke_tokens_issued_before' => before
+                       }
+                     })
```

Comment:
Looks like indentation is not correct here

Reply:
it's automated (rubocop), expected formatting.